src/lib/wallet/GoodWalletProvider.js

```diff
@@ -339,32 +351,94 @@ export const useSwitchNetwork = () => {
   return { switchNetwork, currentNetwork: getNetworkName(goodWallet.networkId) }
 }
 
-export const useSwitchNetworkModal = (toNetwork?: 'fuse' | 'celo', onDismiss = noop) => {
+const NetworkSwitch = withStyles(({ theme }) => ({
```

Comment:
we need a feature flag DELTA_WALLET
any UI changes specific for delta should be under that flag until they are designed+approved by Pat for the wallet.
keep the previous toggle logic when delta_wallet=false

Reply:
done